.golangci.yml

```diff
@@ -73,3 +74,7 @@ linters:
     - whitespace
     - wrapcheck
     - wsl
+    - execinquery
+    # TODO: too much work at this stage, but they should be re-enabled
```

Comment:
Just out of curiosity. What this comment exactly mean?

Reply:
I have improved it a little bit the description

TL;DR; fixing the warning would have required to change a lot of files so I'll open some new issues to track the tasks once this gets merged.